frontend/css/playlists.less

```diff
@@ -71,6 +71,18 @@
   }
 }
 
+.import-playlist-btn {
```

Comment:
I see you are still working on this as these classes aren't used anywhere.
I made a mockup using the existing .add-listen-btn class which does the same thing as this one (a good sign that we can refactor it to use only one), based on the dashboard page "add listen" button (see code ref below):

![image](https://github.com/metabrainz/listenbrainz-server/assets/6179856/42f03fbb-77bf-4c16-986c-9a0ddbc568b7)

https://github.com/metabrainz/listenbrainz-server/blob/c8b5abf56ba77b3b1c81465b01e51b0e144ac6e1/frontend/js/src/user/Dashboard.tsx#L786-L797

Reply:
Thank you very much for your feedback! Wow, there's a lot to work on, but I appreciate all the detailed instructions, such a lifesaver!